docs/customization/grid.rst

```diff
@@ -228,6 +228,240 @@ After these two steps your admin product grid should not have the image field.
 
 .. include:: /customization/plugins.rst.inc
 
+Disabling `fetch join collection` and output walkers
+----------------------------------------------------
+
+In all Grids by default option `fetch_join_collection` and `use_output_walkers` are enabled, if you want to disable it, you should past following code:
```

Comment:
```suggestion
In SyliusGridBundle all grids has options `fetch_join_collection` and `use_output_walkers` enabled by default. According to our research it may fix a lot of pagination issues and improve big-database performance(1M+ rows) up to 70%, but with the price of more queries required to be performed. If this trade-off is not worth it for you, you may disable it by using the following configuration snippet:
```